webpack.config.js

```diff
@@ -2,7 +2,7 @@ var path = require('path');
 var webpack = require('webpack');
 
 module.exports = {
-  devtool: 'eval',
+  devtool: 'cheap-module-eval-source-map',
```

Comment:
A link to what? No specific issue, just Chrome debugger getting messed up.
I recommend `'cheap-module-source-map'` instead.